src/_base/harness/config/confd.yml

```diff
@@ -18,6 +18,7 @@ confd('harness:/'):
   - { src: docker/image/console/root/usr/local/etc/php/conf.d/docker-php-ext-tideways.ini }
   - { src: docker/image/console/root/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini }
   - { src: docker/image/lighthouse/Dockerfile }
+  - { src: docker/image/lighthouse/root/app/run.sh }
```

Comment:
This will need replicating into the `src/akeneo/harness/config/confd.yml` and the rest of the harness `confd.yml`s.